src/my-element.ts

```diff
@@ -471,7 +543,8 @@ export class MyElement extends LitElement {
     graph.drawPriorityLabel('OBJECTIVITY', 225, 'center', textInside, false, kerning)
     graph.drawPriorityLabel('RELIABILITY', 180, 'center', textInside, false, kerning)
 
-
+    this._drawProfileAvatar(180, 2, 1)
+    this._drawProfileAvatar(125, 1.5, 2)
```

Comment:
When drawing, we provide the angle, the vector, and the accountID of the user whose profile we want to draw. The accountID is used to load the correct image.